drivers/sensor/bosch/bme280/bme280.h

```diff
@@ -158,4 +160,60 @@ extern const struct bme280_bus_io bme280_bus_io_i2c;
 					 BME280_TEMP_OVER |  \
 					 BME280_MODE_SLEEP)
 
+struct bme280_data {
+	/* Compensation parameters. */
+	uint16_t dig_t1;
+	int16_t dig_t2;
+	int16_t dig_t3;
+	uint16_t dig_p1;
+	int16_t dig_p2;
+	int16_t dig_p3;
+	int16_t dig_p4;
+	int16_t dig_p5;
+	int16_t dig_p6;
+	int16_t dig_p7;
+	int16_t dig_p8;
+	int16_t dig_p9;
+	uint8_t dig_h1;
+	int16_t dig_h2;
+	uint8_t dig_h3;
+	int16_t dig_h4;
+	int16_t dig_h5;
+	int8_t dig_h6;
+
+	/* Compensated values. */
+	int32_t comp_temp;
+	uint32_t comp_press;
+	uint32_t comp_humidity;
+
+	/* Carryover between temperature and pressure/humidity compensation. */
+	int32_t t_fine;
+
+	uint8_t chip_id;
+};
+
+/*
+ * RTIO
+ */
+
+struct bme280_decoder_header {
+	uint64_t timestamp;
+} __attribute__((__packed__));
+
+struct bme280_encoded_data {
+	struct bme280_decoder_header header;
+	struct bme280_data data;
```

Comment:
This could be the compensated comp_temp, comp_press, comp_humidity without the large number of compensation parameters today